docs/src/gui/gmoccapy.adoc

```diff
@@ -1185,24 +1191,18 @@ toggle DRO mode::
   By default this checkbox is active, so every click on any DRO will toggle the DRO readout from actual to relative to DTG (distance to go). +
   Neverthereless a click on the axis letter will open the popup dialog to set the axis value.
 
-
 .Preview
-
-- _Grid Size_ -  Sets the grid size of the preview window.
+'Grid Size'::
```

Comment:
Please remove this change to fit the style in this document.

Reply:
done.